Gemfile

```diff
@@ -45,10 +45,10 @@ gem 'faraday_middleware'
 gem 'fast_jsonapi'
 gem 'fastimage'
 gem 'figaro'
-gem 'flipper'
-gem 'flipper-active_record'
-gem 'flipper-active_support_cache_store'
-gem 'flipper-ui'
+gem 'flipper', '0.17.1'
```

Comment:
This, to me, declares intent to keep it at version `0.17.1` Can we change this to >=